.github/scripts/get_system_info.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+#  Copyright (C) 2023 Intel Corporation
+#  Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See LICENSE.TXT
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# get-system-info.sh - Script for printing system info
```

Comment:
name here is inconsistent with the file name

Reply:
okey